frappe/public/js/frappe/form/formatters.js

```diff
@@ -50,7 +50,7 @@ frappe.form.formatters = {
 		return frappe.form.formatters._right(value==null ? "" : cint(value), options)
 	},
 	Percent: function(value, docfield, options) {
-		return frappe.form.formatters._right(flt(value, 2) + "%", options)
+		return frappe.form.formatters._right(flt(value, frappe.defaults.get_default("float_precision")) + "%", options);
```

Comment:
```suggestion
		return frappe.form.formatters._right(flt(value, frappe.defaults.get_default("float_precision") || 2) + "%", options);
```

Reply:
@gavindsouza  added || 3 instead of 2 as 2 will reintroduce the problem